docs/news/unreleased/nfs-convert.md

```diff
@@ -0,0 +1,5 @@
+# RHEL 8.4: enable nfs-convert.service
```

Comment:
I wonder if we should mention that this change also affects CentOS Stream... Not sure...

Reply:
Ya I will add that in. I think it makes sense to mention.